src/effects/VST3/VST3Effect.cpp

```diff
@@ -882,7 +894,7 @@ std::unique_ptr<EffectUIValidator> VST3Effect::PopulateUI(ShuttleGui& S,
          auto vSizer = std::make_unique<wxBoxSizer>(wxVERTICAL);
          auto controlsRoot = safenew wxWindow(parent, wxID_ANY);
          if(!LoadVSTUI(controlsRoot))
-            VST3Utils::BuildPlainUI(controlsRoot, mEditController, mComponentHandler);
+            mPlainUI = VST3ParametersWindow::Setup(*controlsRoot, *mEditController, *mComponentHandler);
```

Comment:
I will approve these changes, but there is one remainning suggestion:

When you deference pointers, it's an implied assertion that the pointer is not null.  Better to prove it (with
assertions or invariants) or have null checks.

`controlsRoot` is obviously not null in this context.  Maybe the others need null checks.

Ditto, at the other use of `Setup`